src/main/java/lab5/players/HumanPlayer.java

```diff
@@ -0,0 +1,16 @@
+package players;
+
+import game.Board;
+import game.Position;
+import ui.Console;
+
+public class HumanPlayer extends Player {
+    public HumanPlayer(String name) {
+        super(name);
+    }
+
+    @Override
+    public Position pickNextMove(Board board) {
+        return Console.promptForPosition(board);
```

Comment:
There's more work to be done here. You need to make sure the user selects a non-empty location and re-prompt if they don't

Reply:
Don't forget your JavaDoc to explain what your methods do!